climada/entity/exposures/litpop/litpop.py

```diff
@@ -508,11 +515,11 @@ def set_custom_shape(self, shape, total_value, res_arcsec=30, exponents=(1,1),
             self.meta = {'crs': self.crs}
 
     @staticmethod
-    def _set_one_country(country, res_arcsec=30, exponents=(1,1), fin_mode=None,
+    def _from_country(country, res_arcsec=30, exponents=(1,1), fin_mode=None,
```

Comment:
In this case, an internal method (prefixed with underscore) is a `set_`-method that our new constructor `LitPop.from_countries` relies on. Fortunately, it's only *called* `set_`, but is actually a proper constructor already. We only need to rename the function. In this case, we don't care about backwards compatibility because this is an internal function and not part of the official API.